### PR TITLE
Documentation for allow_empty option in RoutingAuto

### PR DESCRIPTION
**Pull request by dantleech**
Documentation for https://github.com/symfony-cmf/RoutingAuto/pull/39

### CODE REVIEW

File: bundles/routing_auto/token_providers.rst [hunk 1]
```diff
@@ -9,6 +9,15 @@ Such values can be derived form the object for which the route is being
 generated or from the environment (e.g. you could use the current locale in
 the route).
 
+Global options
```
Comment (WouterJ) [inline bundles/routing_auto/token_providers.rst]: "Global Options"

File: bundles/routing_auto/token_providers.rst [hunk 2]
```diff
@@ -9,6 +9,15 @@ Such values can be derived form the object for which the route is being
...
+--------------
+
+The following options apply to all token providers:
+
+``allow_empty``
+  Allow the token value to be empty. If the token value is empty then any
```
Comment (WouterJ) [inline bundles/routing_auto/token_providers.rst]: indentation should be 4 spaces

File: bundles/routing_auto/token_providers.rst [hunk 3]
```diff
@@ -9,6 +9,15 @@ Such values can be derived form the object for which the route is being
...
+  trailing slash will be removed, if the token is the last token, then the
+  leading slash will be removed. Default is ``false``
```
Comment (WouterJ) [inline bundles/routing_auto/token_providers.rst]: missing final stop

File: bundles/routing_auto/token_providers.rst [hunk 4]
```diff
@@ -9,6 +9,15 @@ Such values can be derived form the object for which the route is being
...
 the route).
 
+Global options
+--------------
+
+The following options apply to all token providers:
+
+``allow_empty``
+  Allow the token value to be empty. If the token value is empty then any
+  trailing slash will be removed, if the token is the last token, then the
```
Comment (WouterJ) [inline bundles/routing_auto/token_providers.rst]: "[...] will be removed. If it's the last [...]"

**Comment by dantleech** [comment]
Updated.

**Comment by dbu** [comment]
@WouterJ is it ok to merge this?

**Pull request merged**